example/gluon/lstm_crf.py

```diff
@@ -62,8 +62,9 @@ def __init__(self, vocab_size, tag2idx, embedding_dim, hidden_dim):
 
             # Matrix of transition parameters.  Entry i,j is the score of
             # transitioning *to* i *from* j.
-            self.transitions = nd.random.normal(shape=(self.tagset_size, self.tagset_size))
-
+            self.transitions = gluon.Parameter("crf_transition_matrix", 
```

Comment:
You can use `self.params.get` to declare the parameter so that it doesn't require model.params.update later